### PR TITLE
DP-363

### PR DESCRIPTION
**Pull request by JordanBalesBAO**
### Related Issues
- DP-363
- etc.

## Reviewer:
- [ ] Tests added/updated & passing.
- [ ] Clean linting.
- [ ] Related issue ticket in Jira/GitHub.
- [ ] Documentation added/updated.
- [ ] Code conforms to style guidelines.
- [ ] Well commented.
- [ ] Updates reflected in NEWS.md.
- [ ] Build check passes.

### CODE REVIEW

File: R/data.R [hunk 1]
```diff
@@ -131,134 +151,153 @@
...
+#'   \item{country_name}{Country name associated with the Organisation Unit}
+#'   \item{country_uid}{Country name UID}
+#'   \item{snu1}{Subnational Unit associated with the Organisation Unit}
+#'   \item{snu1_uid}{Subnational Unit UID}
+#'   \item{psnu}{Priority Sub-National Unit associated with the Organisation
+#'    Unit}
+#'   \item{psnu_uid}{Priority Sub-National Unit UID}
+#'   \item{psnu_type}{The type of Priority Sub-National Unit}
+#'   \item{lastUpdated}{The last time the Organisation Unit was updated}
 #'   \item{ancestors}{}
```
Comment (cnemarich) [inline R/data.R:169]: What is the definition for `ancestors` with this dataset?

File: R/data.R [hunk 2]
```diff
@@ -131,134 +151,153 @@
...
 #'   \item{organisationUnitGroups}{}
```
Comment (cnemarich) [inline R/data.R:170]: Could we add a definition for `organisationUnitGroups` here?

File: R/data.R [hunk 3]
```diff
@@ -131,134 +151,153 @@
...
 #' \describe{
-#'   \item{name}{}
-#'   \item{id}{}
-#'   \item{categoryoptiongroup}{}
-#'   \item{datapack_disagg}{}
+#'   \item{name}{Name of the Category Option}
+#'   \item{id}{Category Option UID}
+#'   \item{categoryoptiongroup}{The group by which the category option belongs}
+#'   \item{datapack_disagg}{Name of the Category option in the Data Pack}
 #'   \item{datapack_schema_group}{}
```
Comment (cnemarich) [inline R/data.R:188]: What is the definition of `datapack_schema_group` for this dataset?

File: R/data.R [hunk 4]
```diff
@@ -131,134 +151,153 @@
...
 #'
-#' @description Tibble of data pack names and
+#' @description Tibble of data pack country names and their UID's
 #'
 #' @format
 #' \describe{
-#'   \item{datapack_name}{Name on home tab of datapack}
-#'   \item{country_uids}{countries uids listed on home tab}
+#'   \item{datapack_name}{Country name on home tab of datapack}
+#'   \item{country_uids}{Country's uids listed on the home tab}
```
Comment (cnemarich) [inline R/data.R:200]:
```suggestion
#'   \item{country_uids}{Country's UIDs listed on the home tab}
```

File: R/data.R [hunk 5]
```diff
@@ -131,134 +151,153 @@
...
+#'   \item{datapack_disagg}{Name of the Category option in the Data Pack}
 #'   \item{datapack_schema_group}{}
 #' }
 "valid_category_options"
 
 #' @docType data
 #' @title Datapack country groupings
 #'
-#' @description Tibble of data pack names and
+#' @description Tibble of data pack country names and their UID's
```
Comment (cnemarich) [inline R/data.R:195]:
```suggestion
#' @description Tibble of data pack country names and their UIDs.
```

File: R/data.R [hunk 6]
```diff
@@ -39,18 +41,18 @@
...
+#'   \item{sheet_name}{Lists the sheet/tab name as used in the Data Pack.}
 #'   \item{col}{Value describing the column position of each
 #'   \code{indicator_code}.}
 #'   \item{label}{String label used to describe \code{indicator_code}.}
 #'   \item{indicator_code}{Code used in the Data Pack to uniquely
 #'   identify each distinct programmatic area of target setting.}
-#'   \item{formula}{Excel formula defined for \code{indicator_code}.}
+#'   \item{formula}{Excel formula defined for the \code{indicator_code}.}
 #'   \item{col_type}{Flags whether an \code{indicator_code} is a Target
 #'   (\code{"Target"}), row header (\code{Row Header}) or not (\code{NA}).}
```
Comment (cnemarich) [inline R/data.R:52]:
```suggestion
#'   (\code{target}), historic data (\code{past}), reference figure
#'   (\code{reference}), row header (\code{row_header}) or not (\code{NA}).}
```

File: R/data.R [hunk 7]
```diff
@@ -61,68 +63,86 @@
...
 #'   \item{col}{Value describing the column position of each
 #'   \code{indicator_code}.}
 #'   \item{indicator_code}{Code used in the Data Pack to uniquely
 #'   identify each distinct programmatic area of target setting.}
 #'   \item{dataset}{For \code{indicator_codes} listed as "\code{Targets}"
 #'   in the \code{col_type} field, documents the dataset, either \code{MER},
-#'   \code{IMPATT}, or \code{SUBNAT}.}
+#'   \code{IMPATT},\code{datapack}, or \code{SUBNAT}.}
 #'   \item{col_type}{Flags whether an \code{indicator_code} is a Target
 #'   (\code{"Target"}), row header (\code{Row Header}) or not (\code{NA}).}
```
Comment (cnemarich) [inline R/data.R:133]:
```suggestion
#'   (\code{target}), historic data (\code{past}), reference figure
#'   (\code{reference}), row header (\code{row_header}) or not (\code{NA}).}
```

File: R/data.R (hunk 6 above)
Comment (cnemarich) [inline R/data.R:52]:
```suggestion
#'   (\code{target}), historic data (\code{past}), reference figure
#'   (\code{reference}), row header (\code{row_header}) or not (\code{NA}).}
```

File: R/data.R [hunk 8]
```diff
@@ -303,23 +350,31 @@
...
```
Comment (cnemarich) [inline R/data.R:363]:
```suggestion
#'   (\code{target}), historic data (\code{past}), reference figure
#'   (\code{reference}), row header (\code{row_header}) or not (\code{NA}).}
```

File: R/data.R [hunk 9]
```diff
@@ -269,30 +308,38 @@
...
```
Comment (cnemarich) [inline R/data.R:321]:
```suggestion
#'   (\code{target}), historic data (\code{past}), reference figure
#'   (\code{reference}), row header (\code{row_header}) or not (\code{NA}).}
```

File: R/data.R [hunk 10]
```diff
@@ -131,134 +151,153 @@
...
```
Comment (cnemarich) [inline R/data.R:268]:
```suggestion
#'   (\code{target}), historic data (\code{past}), reference figure
#'   (\code{reference}), row header (\code{row_header}) or not (\code{NA}).}
```

File: R/data.R [hunk 11]
```diff
@@ -131,134 +151,153 @@
...
```
Comment (cnemarich) [inline R/data.R:228]:
```suggestion
#'   (\code{target}), historic data (\code{past}), reference figure
#'   (\code{reference}), row header (\code{row_header}) or not (\code{NA}).}
```

File: R/data.R (hunk 4 above)
Comment (cnemarich) [inline R/data.R:200]:
```suggestion
#'   \item{country_uids}{Country's UIDs listed on the home tab}
```

File: R/data.R (hunk 5 above)
Comment (cnemarich) [inline R/data.R:195]:
```suggestion
#' @description Tibble of data pack country names and their UIDs.
```

File: R/data.R (hunk 3 above)
Comment (cnemarich) [inline R/data.R:188]: What is the definition of `datapack_schema_group` for this dataset?

File: R/data.R (hunk 2 above)
Comment (cnemarich) [inline R/data.R:170]: Could we add a definition for `organisationUnitGroups` here?

File: R/data.R (hunk 1 above)
Comment (cnemarich) [inline R/data.R:169]: What is the definition of `ancestors` for this dataset?

File: R/data.R (hunk 7 above)
Comment (cnemarich) [inline R/data.R:133]:
```suggestion
#'   (\code{target}), historic data (\code{past}), reference figure
#'   (\code{reference}), row header (\code{row_header}), or not (\code{NA}).}
```

**Review by cnemarich — DISMISSED** [review]
I found a few columns in datasets that did not have definitions and one that was improperly documented across datasets (`col_type`) that I have marked. Beyond that, everything looks great.

File: R/data.R (hunk 2 above)
Comment (JordanBalesBAO) [reply R/data.R:172]: Updated with ' A nested two column data frame that
#'    contains the name and id of the groups the organisation unit is associated
#'    with. For example "Community" and "PvuaP6YALSA" '

**Comment by sam-bao** [comment]
@JordanBalesBAO when you update documentation please remember to also build the documentation with `devtools::document()`. I completed this in the last commit.

**Pull request merged**